packages/network/test/unit/pki_spec.ts

```diff
@@ -0,0 +1,178 @@
+import { Url } from 'url'
```

Comment:
there seems to have been some kind of error here - this is not a spec, this is a line-for-line copy of `packages/network/lib/pki.ts`, this should be replaced with unit tests

Reply:
I'm told the tests do exist, were omitted by mistake, and they'll be submitted soon!

Reply:
@flotwig Are you happy this is resolved in the latest commit? Can this conversation be resolved?